sotodlib/preprocess/processes.py

```diff
@@ -68,6 +69,42 @@ def select(self, meta, proc_aman=None):
         meta.restrict("dets", meta.dets.vals[has_all_cut(keep)])
         return meta
 
+    @classmethod
```

Comment:
this isn't actually a class method as it doesn't generate a class instance? Maybe you want this to be a static method?

Reply:
Looking into python/code things and how you're using it. This is a bit of a non-standard use of a classmethod (at least how I see it the most). 

Are we ever going to be running multiple pipelines at once? If so the influx-name probably shouldn't be a class variable and should instead be defined through a config file. 

I think we want to include telescope in the `tags` because tel-tube is not unique between the SATs

Reply:
Sorry I realize I'm mixing nomenclatures. tel-tube is from imprinter and is unique. But I think telescope (`meta.obs_info.telescope`) is another tag you should add to this data.

Reply:
I'll add the telescope tag.

re `classmethod`: I'm using it here because I want to access a class attribute. I think this is a normal use case, but I'd be curious to know if this is discouraged in any references.

Reply:
That's a good question about running multiple pipelines... I thought putting the name in the class definition might be a good way of keeping it standard. Maybe if there are parallel pipelines they should be recording to sepaprate databases?